test/tests/selftest/pbs_pbstestsuite.py

```diff
@@ -268,3 +268,18 @@ def test_revert_pbsconf_fewer_vars(self):
         # Confirm that the variable was set again
         pbs_conf_val = self.du.parse_pbs_config(self.server.hostname)
         self.assertIn("PBS_CORE_LIMIT", pbs_conf_val)
+
+    def test_revert_moms_default_conf(self):
+        """
+        Test if PBSTestSuite.revert_moms() reverts the mom configuration
+        setting to defaults
+        """
+        a = {'$prologalarm': 280}
+        self.mom.add_config(a)
+
+        c1 = {'$prologalarm': '280',
+              '$clienthost': self.server.hostname}
+        self.assertEqual(self.mom.parse_config(), c1)
```

Comment:
Is it possible the mom config would have something other than `$clienthost` in the original config?
I would store the dict of `self.mom.parse_config()` before adding the `$prologalarm` option and comparing it with the next one.